src/autocomplete/databases/redis/tests/increment/increment.test.ts

```diff
@@ -0,0 +1,19 @@
+import {parseRedisQueryWithoutCursor} from '../../index';
+
+test('should not report errors on INCR command', () => {
+    const autocompleteResult = parseRedisQueryWithoutCursor('INCR test');
+
+    expect(autocompleteResult.errors).toHaveLength(0);
+});
+
+test('should not report errors on multiple INCR commands', () => {
+    const autocompleteResult = parseRedisQueryWithoutCursor('INCR test\nINCR 123');
```

Comment:
We split queries by newline? Maybe it's better to force `;` in the end?

Reply:
we can't use semicolon, because it is valid to use in key/value name, e.g. `SET key value;` will set value to `value;`
that's why Redis commands are separated by newline, that's also the way they do it in DataGrip